convert2rhel/unit_tests/checks_test.py

```diff
@@ -625,6 +631,18 @@ def test_check_efi_efi_detected_ok(self):
         self._check_efi_detection_log()
         self.assertEqual(len(checks.logger.warning_msgs), 0)
 
+    @unit_tests.mock(grub, "is_efi", lambda: True)
+    @unit_tests.mock(grub, "is_secure_boot", lambda: False)
+    @unit_tests.mock(checks.system_info, "arch", "x86_64")
+    @unit_tests.mock(checks.system_info, "version", _gen_version(7, 9))
+    @unit_tests.mock(checks, "logger", GetLoggerMocked())
+    @unit_tests.mock(os.path, "exists", lambda x: x == "/usr/sbin/efibootmgr")
+    @unit_tests.mock(grub, "EFIBootInfo", EFIBootInfoMocked(current_bootnum="000A"))
+    def test_check_efi_efi_detected_ok_hex_entry(self):
+        checks.check_efi()
+        self._check_efi_detection_log()
+        self.assertEqual(len(checks.logger.warning_msgs), 0)
```

Comment:
There's not enough assertions here to make sure that it detected EFI correctly. If you revert the RegEx and run the tests it will give a false positive since there is at least one valid boot detected.

You could assert that the entries contain a `EFIBootLoader` with the `000A` entry. Or better yet, change the added entries to only have one Centos Linux with entry `000A` instead of the two that's there right now

Reply:
I have added a few more asserts, let me know what you think about that change

Reply:
Just looked a bit closer at the test. We're testing the wrong thing here, we should be testing that the `EFIBootInfo` parses everything correctly in `grub_test.py@test_efibootinfo`. Right now you've added mock with the values already set so no parsing will take place

Reply:
Oh, that's right! The current test in checks_test.py is not useful at all. I've updated the right one now.

Reply:
Yep! I'd say we can remove this test completely

Reply:
Done!